lib/get/onValue.js

```diff
@@ -34,6 +34,11 @@ module.exports = function onValue(model, node, seed, depth, outerResults,
         valueNode = clone(node);
     }
 
+    // We don't want to emit references in json output
+    else if (!isJSONG && node.$type === $ref) {
+        valueNode = undefined;
```

Comment:
That was the original idea (and is simple to implement) but after a discussion with @jhusain yesterday the goal is to omit keys that are missing, atom of undefined, or ref.